.cirrus.yml

```diff
@@ -151,6 +175,48 @@ task:
   install_script:
     - brew install coreutils
 
+  libs_cache:
+    folder: build/libs
+    fingerprint_script: echo "`md5 lib/CMakeLists.txt` macos"
+    populate_script: make libs build_flags=-j8 verbose=1
+
+  configure_script:
+    - make configure verbose=1
+  build_script:
+    - make build build_flags=-j8 verbose=1
+  test_script:
+    - make test-ci verbose=1
+
+  only_if: $CIRRUS_BRANCH != 'master' && $CIRRUS_BRANCH != 'release' && $CIRRUS_API_CREATED != "true"
+
+task:
+  only_if: $CIRRUS_PR != ''
+
+  windows_container:
+    dockerfile: .ci-dockerfiles/windowsservercore/Dockerfile
+    os_version: 2019
+    cpu: 8
+    memory: 24
+
+  name: "PR: Windows X64"
+
+  timeout_in: 120m
```

Comment:
wow, is this needed? how slow are the builds?

with 8 cpus, i'm surprise we need to change the timeout.

Reply:
I just added the 8 CPU line; I'm hoping it will take a lot less time, since before it was only 2.

Reply:
If you can parallelize the build, it should have a large impact. I assume that it works for windows machines. It would do nothing for mac builds because all the machines are only 2 cpus.

Reply:
MSVC tries to use as many cores as it can.

Reply:
With 2 cores, the Linux LLVM builds would take about 80-100 minutes. They are down to 20-30 (including cloning) with parallelization. Hopefully, the Windows build sees a similar improvement.